config/params/mainnet_config.go

```diff
@@ -16,6 +16,16 @@ func MainnetConfig() *BeaconChainConfig {
 	return mainnetBeaconConfig
 }
 
+func ComputeGenesisValidatorsRoot(s string) [32]byte {
```

Comment:
why not just hardcode everything as [32]byte instead of implementing this function